pkg/lightning/mydump/csv_parser.go

```diff
@@ -398,7 +398,7 @@ func (parser *CSVParser) readUntil(chars *byteSet) ([]byte, byte, error) {
 	var buf []byte
 	for {
 		buf = append(buf, parser.buf...)
-		if len(buf) > LargestEntryLimit {
+		if parser.checkRowLen && parser.pos-parser.rowStartPos+int64(len(buf)) > int64(LargestEntryLimit) {
```

Comment:
in all cases, `parser.checkRowLen` is true when running here?

Reply:
only public API check it, i.e. ReadRow/ReadColumns/ReadUntilTerminator